content/blog/posts/c-stl-algorithms-series-part-1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "C++ STL Algorithms Series - Part 1"
+date: 2022-07-13
+slug: c-stl-algorithms-series-part-1
+author: Tom Hulton-Harrop
+blog_img: ""
+full_img: ""
+---
+# Item 1: C++ STL Algorithms Series - Part 1
```

Comment:
```suggestion
## C++ STL Algorithms Series - Part 1
```
Not sure if this is required, the title may automatically be included in the page.